### PR TITLE
Implemented to parse properly regarding resource bundle usage of C++ files

### PR DESCRIPTION
**Pull request by gouniLee**
Many of webOS services are written c or cpp program. I create this module to extract the localization string properly in C++ files. Here's API spec & example. Most implementation is similar to a module `ilib-loctool-webos-c`

```
const string& ResBundle::getLocString(const string& source)
const string& ResBundle::getLocString(const string& key, const string& source)
```

```
m_localeStringTryControl = mp_resBundle->getLocString("The external device is requesting TV control."); // i18n
m_localeStringYes = mp_resBundle->getLocString("YES"); // i18n // photovideo-48
```

comment style:
```
/* i18n hello */
// i18n  hello
```

### CODE REVIEW

File: CppFile.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    var trimComment = commentString.
+        replace(/\s*\*\//, "").
+        replace(/\s*\:\s*/, "").
+        replace(/\/*\s*\**\s*/, "").
+        replace(/\s*\*\s*/, "");
+    return trimComment;
+}
+
+var reGetLocString = new RegExp(/\bgetLocString\("((\\"|[^"])*)"\s*\)/g);
+var reGetLocStringWithKey = new RegExp(/\bgetLocString\(\s*"(.[^);]*)"\s*,\s*"(.[^);]*)"(\s*\))/g);
```
Comment (ehoogerbeets) [inline CppFile.js]: This regexp should extract the strings in the same way that the first one does

File: test/testCppFile.js [hunk 2]
```diff
@@ -0,0 +1,840 @@
...
+            type: cppft
+        });
+        test.ok(cppf);
+
+        // should read the file
+        cppf.extract();
+        var set = cppf.getTranslationSet();
+        test.equal(set.size(), 3);
+
+        var r = set.getBySource("Your\n Welcome");
```
Comment (ehoogerbeets) [inline test/testCppFile.js]: Minor nitpick: it should be "You're Welcome". You're = You are. Your = possessive.

**Review by ehoogerbeets — APPROVED** [review]

**Comment by ehoogerbeets** [comment]
These should be published on npm when they are ready.

File: test/testCppFile.js (hunk 2 above)
Comment (gouniLee) [reply test/testCppFile.js]: Oh, you're right. Thanks for the correction :)

**Comment by gouniLee** [comment]
Yep, Both `ilib-loctool-webos-c` and `ilib-loctool-webos-cpp` modules will be published to npm soon.

**Pull request merged**